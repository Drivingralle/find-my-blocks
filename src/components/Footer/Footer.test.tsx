import React from "react";
import renderer from "react-test-renderer";
import { Footer } from "./";

it("render correctly", () => {
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
});
