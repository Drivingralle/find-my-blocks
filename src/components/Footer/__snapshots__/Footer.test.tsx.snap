// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<div
  className="footer"
>
  Developed by
   
  <a
    className="link"
    href="https://eddysims.com"
    target="_blank"
  >
    Eddy Sims
  </a>
  <a
    className="link"
    href="https://find-my-blocks.eddysims.com/donate"
    target="_blank"
  >
    <svg
      className="feather feather-heart "
      fill="none"
      height={24}
      stroke="currentColor"
      strokeLinecap="round"
      strokeLinejoin="round"
      strokeWidth="2"
      viewBox="0 0 24 24"
      width={24}
    >
      <g
        dangerouslySetInnerHTML={
          Object {
            "__html": "<path d=\\"M20.84 4.61a5.5 5.5 0 0 0-7.78 0L12 5.67l-1.06-1.06a5.5 5.5 0 0 0-7.78 7.78l1.06 1.06L12 21.23l7.78-7.78 1.06-1.06a5.5 5.5 0 0 0 0-7.78z\\"></path>",
          }
        }
      />
    </svg>
    Make a donation
  </a>
</div>
`;
